Platform/OpenCanopy/OpenCanopy.c

```diff
@@ -1356,12 +1493,11 @@ GuiGetInterpolatedValue (
   UINT32 DeltaTime;
 
   ASSERT (Interpol != NULL);
-  ASSERT (Interpol->StartTime <= CurrentTime);
   ASSERT (Interpol->Duration > 0);
 
   STATIC CONST UINT32 InterpolFpTimeFactor = 1U << 12U;
 
-  if (CurrentTime == Interpol->StartTime) {
+  if (CurrentTime <= Interpol->StartTime) {
```

Comment:
@Download-Fritz, is this correct?

Reply:
It's to support sequential animations: one animation starts after the end of the previous one, so StartTime can be not 0.